book.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# The Book class represents a book, which has a title, author, and may be rented by people.
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author
+end
```

Comment:
- [ ] Please define a method for adding `rentals` in order tot fulfil the `has-many` requirement as shown below 👇 
![image](https://user-images.githubusercontent.com/46839548/225501044-4abf386d-766e-48e9-8f59-63645f493809.png)